browser/brave_wallet/external_wallets_importer_unittest.cc

```diff
@@ -104,21 +104,6 @@ class ExternalWalletsImporterUnitTest : public testing::Test {
                                           base::JSON_PARSE_CHROMIUM_EXTENSIONS);
     base::expected<ImportInfo, ImportError> out_info;
 
-    {
-      ExternalWalletsImporter importer(mojom::ExternalWalletType::CryptoWallets,
-                                       browser_context());
-
-      importer.SetStorageDataForTesting(json.Clone());
-
-      base::RunLoop run_loop;
-      importer.GetImportInfo(
-          password, base::BindLambdaForTesting(
-                        [&](base::expected<ImportInfo, ImportError> info) {
-                          out_info = std::move(info);
-                          run_loop.Quit();
-                        }));
-      run_loop.Run();
```

Comment:
Should we also drop `DISABLED_ImportLegacyWallet` test referencing `valid_legacy_mnemonic` and `valid_legacy_data`?
cc: @darkdh

Reply:
ah, yes because we no longer have that route in the wallet importer

Reply:
Removed in 96583cd1381657e665f5826d087ee0811f7de66a